.travis.yml

```diff
@@ -1,16 +1,19 @@
 language: objective-c
+osx_image: xcode7.1
 env:
-global:
-    - "MODULE_NAME=ti.barcode"
+ global:
+   - "ARTIFACTS_AWS_REGION=us-east-1"
+   - "ARTIFACTS_S3_BUCKET=builds.appcelerator.com"
+   - "MODULE_NAME=ti.barcode"
 before_install:
     - MODULE_ROOT=$PWD
 install:
     - cd $MODULE_ROOT
-    - curl -o install.sh https://raw.githubusercontent.com/appcelerator-modules/ci/master/travis/install.sh
-    - source install.sh 
-script: 
+    - curl -o install.sh https://raw.githubusercontent.com/appcelerator-modules/ci/master/travis/install.sh #change this to appcelerator-modules once PR has been merged
+    - source install.sh -s "--branch 5_1_X"
```

Comment:
I changed this to `- source install.sh -a 22 -s "5.1.2.v20151216190036" ` in https://github.com/appcelerator-modules/ti.barcode/pull/33

The Android part is passing with that change. But overall it's still failing.